checkout/access-checkout-android/src/main/java/com/worldpay/access/checkout/api/serialization/LinkDiscoveryDeserializerFactory.kt

```diff
@@ -0,0 +1,7 @@
+package com.worldpay.access.checkout.api.serialization
+
+internal class LinkDiscoveryDeserializerFactory() {
```

Comment:
I don't think this factory is needed, you should be able to pass in the deserialiser into the client. You should move the namespace into the discover method as a parameter so that we can use a single instance of it and it can still be tested.

Reply:
Removed and endpoint class added

Reply:
Removed factory and added endpoint class